libevmasm/Assembly.cpp

```diff
@@ -41,10 +41,16 @@ void Assembly::append(Assembly const& _a)
 	auto newDeposit = m_deposit + _a.deposit();
 	for (AssemblyItem i: _a.m_items)
 	{
-		if (i.type() == Tag || i.type() == PushTag)
+		switch (i.type())
+		{
+		case Tag:
+		case PushTag:
 			i.setData(i.data() + m_usedTags);
-		else if (i.type() == PushSub || i.type() == PushSubSize)
+			break;
+		case PushSub:
+		case PushSubSize:
 			i.setData(i.data() + m_subs.size());
+		}
```

Comment:
This should show an error because not all cases of the enum are handled in the switch..

Reply:
:smiley: Thanks @axic Fixed the issue.